README.md

```diff
@@ -49,10 +49,10 @@ We use [pylint](https://pylint.org) as Python linter.
 
 ## Coding Style
 
-In general, we follow PEP8 Python coding style and Google pyguide
-However, if PEP8 and Google pyguide has conflict style, PEP8 take place.
+In general, we follow [PEP 8](https://www.python.org/dev/peps/pep-0008/) Python coding style and Google pyguide
+However, if [PEP 8](https://www.python.org/dev/peps/pep-0008/) and [Google Python Style Guide](http://google.github.io/styleguide/pyguide.html) have conflict style, [PEP 8](https://www.python.org/dev/peps/pep-0008/) take place.
```

Comment:
Should we change `take` to `takes`?
`[PEP 8](https://www.python.org/dev/peps/pep-0008/) takes place.`

Reply:
Also, can be rephrased to 
`However, when [PEP 8](https://www.python.org/dev/peps/pep-0008/) is in conflict with [Google Python Style Guide](http://google.github.io/styleguide/pyguide.html), [PEP 8](https://www.python.org/dev/peps/pep-0008/) should be given precedence.`

Reply:
Good point! Resolved.